src/parser/preprocessor.c

```diff
@@ -4064,7 +4089,7 @@ try_input:;
 			if (current_call.argument_brace_count > 0) {
 				--current_call.argument_brace_count;
 			} else {
-				assert(kind == ')');
+				assert(current_call.argument_brace_count == 0);
```

Comment:
Is this change necessary for the _Pragma support or unrelated?

Reply:
no it's not!, the assertion is redundant as you can see in the if condition leading to the current branch. i thought this was the original intention from the context.